statistics/table.go

```diff
@@ -487,9 +504,16 @@ func (coll *HistColl) getEqualCondSelectivity(idx *Index, bytes []byte, usedCols
 			}
 			ndv = mathutil.MaxInt64(ndv, coll.Columns[colID].NDV)
 		}
-		return outOfRangeEQSelectivity(ndv, coll.ModifyCount, int64(idx.TotalRowCount()))
+		return outOfRangeEQSelectivity(ndv, coll.ModifyCount, int64(idx.TotalRowCount())), nil
 	}
-	return float64(idx.CMSketch.QueryBytes(bytes)) / float64(idx.TotalRowCount())
+	idxCount := float64(idx.CMSketch.QueryBytes(bytes))
+	var count float64
+	if idxCount < minRowCount {
+		count = idxCount
+	} else {
+		count = minRowCount
```

Comment:
`minRowCount` is an upper bound. For example, for `a = 1 and b = 2`, we'd better use `(count_a / idx.TotalRowCount()) * (count_b / idx.TotalRowCount())` as the final selectivity, instead of `min(count_a, count_b) / idx.TotalRowCount()`.

Reply:
```
if minRowCount < `float64(idx.CMSketch.QueryBytes(bytes))` {
   (count_a / idx.TotalRowCount()) * (count_b / idx.TotalRowCount())
} else {
    float64(idx.CMSketch.QueryBytes(bytes)) / idx.TotalRowCount()
}
```

Is this ok?

Reply:
> > Please add unit tests for this PR as well.
> 
> @eurekaka Can you tell me more detail about how to create some unit tests for this case?

You can manufacture a table with composite index and data rows, then analyze this table and dump its json stats file out by `curl http://127.0.0.1:10080/stats/dump/db_name/table_name`. Then you find the CMSketch for the composite index in the dumped stats, and modify the row counts for all cells to a very large value. Then you can reference `TestIndexEqualUnknown` to load the modified stats and check the selectivity computed using the modified stats.